nomad/csi_endpoint.go

```diff
@@ -120,15 +120,19 @@ func (v *CSIVolume) List(args *structs.CSIVolumeListRequest, reply *structs.CSIV
 			if err != nil {
 				return err
 			}
+			// TODO: handle prefix arg
+
```

Comment:
```suggestion
```

Reply:
assuming this is sufficiently handled below?

Reply:
Whoops! Note left to myself there.